src/ChooseThemeComponent/ChooseTheme.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+    font-family: "KorolevLight";
+    src: local("KorolevLight"),
+     url("../fonts/Korolev-Light.woff") format("woff");
+    font-weight: bold;
+  }  
+  
+  @font-face {
+    font-family: "KorolevHeavy";
+    src: local("KorolevHeavy"),
+     url("../fonts/Korolev-Heavy.woff") format("woff");
+    font-weight: normal;
+  }  
+  
+p {
+
+    font-family: KorolevLight;
+    font-style: normal;
+    font-weight: 500;
+    /*font-weight: 500;*/
+    text-align: center;
+    color: #FFFFFF;
+}
+
+
+
+.main-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 40px;
+    padding-bottom: 20px;
+    width: 1361px;
+    height: 406px;
+
+    background: #F2F2F2;
+    border-radius: 20px;
+
+}
+
+.carousel-item-container{
+    height: 100%;
+    padding: 15px;
+    justify-content: center;
+}
+
+/* .rec.rec-dot {
+    background: #141414;
+}
+
+.rec.rec-dot_active{
+    background: #f2f2f2;
+
+} */
+
+
+.option {
+    border-radius: 20px;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    display: block;
+    padding: 5px;
+}
+
+.option-btn{
+    display: block;
+    position: center;
+    margin-left: auto;
+    margin-right: auto;
+    width: 140.43px;
+    height: 49.99px;
```

Comment:
This is good, but I wanted to share a more industry-standard way of setting the size of elements. Nowadays the industry prefers using rem over px for many reasons. Please refer to the link provided below.

https://uxdesign.cc/why-designers-should-move-from-px-to-rem-and-how-to-do-that-in-figma-c0ea23e07a15#:~:text=rem%20is%20a%20relative%20unit,rem%20actually%20stands%20for%20root).&text=REM%20does%20not%20translate%20into,whole%20point%20of%20using%20it.

Reply:
Interesting. Thank you for the info! I just copied what was on the figma I think which was originally in px. But maybe there is a way to change what I import from figma from px to rem.

Reply:
The general rule for converting px to rem most of the time is deleting it by 16. example 140px would be 140/16 so 8.75rem.

In rare cases, the number is not 16 but for now, you can treat the conversion to be 1rem = 16px